### PR TITLE
Pod create alert fires more than once

### PR DESCRIPTION
**Pull request by n313893254**
https://github.com/rancher/rancher/issues/13517

### CODE REVIEW

File: lib/alert/addon/components/alert/time-duration/component.js [hunk 1]
```diff
@@ -20,4 +20,13 @@ export default Component.extend({
 
     set(this, 'value', h * 3600 + m * 60 + s);
   }.observes('h', 'm', 's'),
+
+  setTime: function() {
+    const value = get(this, 'value') || 0;
+    const d = moment.duration(value * 1000);
+
+    set(this, 'h', d.hours());
```
Comment (westlywright) [inline lib/alert/addon/components/alert/time-duration/component.js]: The following three sets should use this helper method `setProperties(this, { h: ..., ...})`

File: lib/alert/addon/components/alert/time-duration/component.js [hunk 2]
```diff
@@ -20,4 +20,13 @@ export default Component.extend({
...
+    set(this, 'm', d.minutes());
+    set(this, 's', d.seconds());
+  }.observes('value'),
```
Comment (westlywright) [inline lib/alert/addon/components/alert/time-duration/component.js]: Change this to use the `observer` import

**Comment by n313893254** [comment]
Fixed

**Comment by n313893254** [comment]
Fixed

**Pull request merged**